EIPS/eip-4844.md

```diff
@@ -146,20 +146,10 @@ In the transaction-pool and local transaction-journal the network encoding is us
 
 For previous types of transactions the network encoding is no different, i.e. `TransactionNetworkPayload == TransactionPayload`.
 
-The `TransactionNetworkPayload` wraps a `TransactionPayload` with additional data:
-this wrapping data SHOULD be verified directly before or after signature verification.
-
-When a blob transaction is passed through the network (see the [Networking](#networking) section below),
-the `TransactionNetworkPayload` version of the transaction also includes `blobs` and `kzgs` (commitments list).
-The execution layer verifies the wrapper validity against the inner `TransactionPayload` after signature verification as:
-
-- All hashes in `blob_versioned_hashes` must start with the byte `BLOB_COMMITMENT_VERSION_KZG`
```

Comment:
I suggest unifying the naming of `BLOB_COMMITMENT_VERSION_KZG` (in EIP) with `VERSIONED_HASH_VERSION_KZG` (in CL specs)